build/configs/artik055s/Make.defs

```diff
@@ -73,6 +76,9 @@ else
   ARCHINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../framework/include -isystem $(TOPDIR)/../external/include
   ARCHINCLUDES += -isystem $(TOPDIR)/net/lwip/src/include
   ARCHXXINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../external/include
+ifeq ($(CONFIG_AUDIO),y)
+  ARCHXXINCLUDES = -I. -isystem $(TOPDIR)/include -isystem $(TOPDIR)/../framework/include -isystem $(TOPDIR)/../external/include -isystem $(TOPDIR)/../external/include/libcxx
+endif
```

Comment:
Do we need `CONFIG_AUDIO`? I think it is just added without any conditioanl. And duplicated almost.